.gitignore

```diff
@@ -10,7 +10,6 @@ test-results/
 playwright-report/
 Sources/ContentScopeScripts/dist/
 test-results
-!Sources/ContentScopeScripts/dist/pages/.gitignore
```

Comment:
This file doesn’t seem to exist, and we’re already ignoring the parent directory in `Sources/ContentScopeScripts/dist/`. I don’t think this lines does anything